### PR TITLE
chore(main): switch over to main

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

**Review by mtomcal — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Install dependencies
+        run: npm ci
+      - name: Release
+        env:
+          GIT_AUTHOR_EMAIL: ${{ secrets.GIT_AUTHOR_EMAIL }}
+          GIT_AUTHOR_NAME: ${{ secrets.GIT_AUTHOR_NAME }}
+          GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
+          GIT_COMMITTER_NAME: ${{ secrets.GIT_COMMITTER_NAME }}
+          GITHUB_TOKEN: ${{ secrets.PA_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/release.yml:36]: We should double check we have the secrets in place.

**Review by Francois-Esquire — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        node-version: ${{ matrix.node }}
+    - name: Install Dependencies
+      run: npm ci
+      env:
+        NODE_ENV: development
+    - name: Unit Tests
+      run: npm run test:unit
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:32]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Git History Test
+      run: npm run test:git-history
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:37]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Lockfile Lint Test
+      run: npm run test:lockfile
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:42]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Lint
+      run: npm run test:lint
+      env:
+        NODE_ENV: production
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Francois-Esquire) [inline .github/workflows/tests.yml:47]:
```suggestion
```

**Review by Francois-Esquire — CHANGES_REQUESTED** [review]
Small security concern with the token...

File: .github/workflows/release.yml (hunk 1 above)
Comment (JamesSingleton) [reply .github/workflows/release.yml:36]: Yeah I will need @nellyk to add these

**Review by Francois-Esquire — APPROVED** [review]

**Pull request merged**